f-lab-edu/mss-data-pipeline
# 🔀 :: Perf/crawling

## Describe your changes
- single crawler: make this return only goods' information, given detail page url
- list crawler: make this return only goods' url, given list page url
- save to dw: in this file crawling task run
- etc: fix trivial matters, do chore

## what is your changes needed for 
combine crawler modules

## How has this been tested?
- run insert query and check the query runs well in DW(postgres)
- run save_to_dw.py and check the script runs well
